### ISSUE TITLE
seeker.py: error: unrecognized arguments: -t manual


### ISSUE THREAD

**Issue by bro-kengrl0810** [thewhiteh4t/seeker#398]
I'm trying to install and use seeker but I get this error.. seeker.py: error: unrecognized arguments: -t manual
I've tried a few different instructions on. Installing and keep getting the 
**To Reproduce**

Steps to reproduce the behavior, we cannot fix the issue without 
~/seeker $ git clone https://github.com/thewhiteh4t/seeker.git
~/seeker $ ls
~/seeker $ bash install.sh
~/seeker $ python seeker.py -t manual

**Expected behavior**

A clear description of what you expected to happen.

---

**Screenshots**

Add screenshots to help explain your problem.

---

**Please complete the following information :**
 
 - OS:  Android 
 - OS Version Android 12
 - Browser chrome
 - Full Browser Version 103.0.5060.129

I am new to this so I don't understand alot yet.

**Comment by thewhiteh4t** [comment]
@bro-kengrl0810 read usage instructions

**Comment by Anub12** [comment]
yeah, I think - t manual is now removed from seeker, read the instructions below.

**Comment by brok-en0810** [comment]
I got it thanx.. now I'm just having the issue of every repo I try it fails
for authentification..
